lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/driver/TaskStatusCheckPeriod.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.runime.azbatch.driver;
+
+import org.apache.reef.tang.annotations.Name;
+import org.apache.reef.tang.annotations.NamedParameter;
+
+/**
+ * The period to check for task status from Azure batch in ms.
+ */
+@NamedParameter(doc = "The evaluator heartbeat period in ms.", default_value = "1000")
+public final class TaskStatusCheckPeriod implements Name<Integer> {
```

Comment:
Let's rename this to AzureBatchTaskStatusCheckPeriod to avoid confusing it with REEF tasks.

Reply:
Done.